### PR TITLE
Add metric for current task-runner version

### PR DESCRIPTION
**Pull request by thomshutt**

### CODE REVIEW

File: cmd/task-runner/task-runner.go [hunk 1]
```diff
@@ -143,6 +144,9 @@ func Run(build BuildFlags) {
 	cli := parseFlags(build)
 	glog.Infof("Task runner starting... version=%q", build.Version)
 
+	// Fire a metric to track which version of task-runner we're running
+	metrics.Version.WithLabelValues("task-runner", build.Version).Inc()
```
Comment (thomshutt) [inline cmd/task-runner/task-runner.go:148]: @victorges open to moving this out of `Run()`, it was just less obvious where to put it within the `metrics` package

File: metrics/http.go [hunk 2]
```diff
@@ -29,6 +29,13 @@ var (
 		},
 		[]string{"api"},
 	)
+	Version = Factory.NewCounterVec(
+		prometheus.CounterOpts{
```
Comment (hjpotter92) [inline metrics/http.go:33]: should it be counter or gauge? (few suggestions only stated gauge is better for one-off info). redis also uses gauge to export its version/deployment info as metrics

Comment (thomshutt) [reply metrics/http.go:33]: Maybe! I'll switch it, since I have no idea

File: cmd/task-runner/task-runner.go (hunk 1 above)
Comment (victorges) [reply cmd/task-runner/task-runner.go:148]: Oh I thought there was a "high level" initialization function on the package, but maybe not in this project as the metric is also declared statically. If there's no such thing as a `metrics.Init` or `metrics.NewX`, this looks good

**Review by victorges — APPROVED** [review]

File: cmd/task-runner/task-runner.go (hunk 1 above)
Comment (thomshutt) [reply cmd/task-runner/task-runner.go:148]: Yeah, we don't have any metrics initialisation function at the moment, so will leave it here for now

**Pull request merged**